src/coreclr/jit/buildstring.cpp

```diff
@@ -1,17 +1,13 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-#define STRINGIFY(L)     #L
-#define MAKESTRING(M, L) M(L)
-#define STRINGIZE(X)     MAKESTRING(STRINGIFY, X)
-
 #if defined(__clang__)
 #define BUILD_COMPILER                                                                                                 \
-    "Clang " STRINGIZE(__clang_major__) "." STRINGIZE(__clang_minor__) "." STRINGIZE(__clang_patchlevel__)
+    "Clang " STRINGIFY(__clang_major__) "." STRINGIFY(__clang_minor__) "." STRINGIFY(__clang_patchlevel__)
```

Comment:
What definition of `STRINGIFY` is being picked up here? Can you please add a proper include for it?